docker/pbstarphase/Dockerfile

```diff
@@ -12,9 +12,9 @@ ARG CPIC_BUILD_VERSION
 RUN wget https://github.com/PacificBiosciences/pb-StarPhase/archive/refs/tags/v${PBSTARPHASE_VERSION}.tar.gz \
 	&& tar --no-same-owner -zxvf v${PBSTARPHASE_VERSION}.tar.gz --directory /opt \
 	&& rm v${PBSTARPHASE_VERSION}.tar.gz \
-	&& ln -s /opt/pb-StarPhase-${PBSTARPHASE_VERSION}/data/${CPIC_BUILD_VERSION} /opt/pbstarphase_db.json.gz \
-	&& wget https://github.com/PacificBiosciences/pb-StarPhase/releases/download/v${PBSTARPHASE_VERSION}/pbstarphase-v${PBSTARPHASE_VERSION}-x86_64-unknown-linux-gnu.tar.gz \
+	&& ln -s /opt/pb-StarPhase-${PBSTARPHASE_VERSION}/data/${CPIC_BUILD_VERSION} /opt/pbstarphase_db.json.gz
+ARG wget https://github.com/PacificBiosciences/pb-StarPhase/releases/download/v${PBSTARPHASE_VERSION}/pbstarphase-v${PBSTARPHASE_VERSION}-x86_64-unknown-linux-gnu.tar.gz \
```

Comment:
should this be `RUN` instead of `ARG`

Reply:
Yes, and I'm not even sure why this is working.